lib/Conversion/TorchToLinalg/Pooling.cpp

```diff
@@ -1665,4 +2125,5 @@ void mlir::torch::torch_to_linalg::populatePoolingPatternsAndLegality(
       typeConverter, context);
   patterns.add<ConvertAtenAdaptivePoolOp<AtenAdaptiveMaxPool3dOp>>(
       typeConverter, context);
+  patterns.add<ConvertRoiAlignOp>(typeConverter, context);
```

Comment:
You can add all of the patterns with the standard constructor to the same `add` call:
```c++
patterns.add<P0, P1, P2, P3>(typeConverter, context);
```